src/business/AccountService.java

```diff
@@ -0,0 +1,26 @@
+package business;
+
+
+import dataaccess.AccountRepository;
+
+
+public class AccountService {
+	AccountRepository acountRepo = new AccountRepository();
+	public boolean validateUser(String username, String password) throws LibrarySystemException {
```

Comment:
`LibrarySystemException` is never thrown inside method so remove it from method signature

Reply:
right , now the method is removed.